airbyte-workers/src/main/java/io/airbyte/workers/temporal/check/connection/CheckConnectionActivity.java

```diff
@@ -15,8 +15,8 @@
 public interface CheckConnectionActivity {
 
   @ActivityMethod
-  StandardCheckConnectionOutput run(JobRunConfig jobRunConfig,
-                                    IntegrationLauncherConfig launcherConfig,
-                                    StandardCheckConnectionInput connectionConfiguration);
+  StandardCheckConnectionOutput check(JobRunConfig jobRunConfig,
```

Comment:
I would be very careful with the renaming. the call to it should be managed using the temporal versions.

Reply:
The rename came from https://github.com/airbytehq/airbyte/pull/12425 with the note: 

> this rename was necessary, otherwise we were trying to register two activities whose entrypoint was called run which confused temporal

I do think this implies that workflows which use the activity should be version bumped, which I *think* I did.